src/Spatial/Units/Angle.cs

```diff
@@ -311,6 +335,44 @@ public static Angle ReadFrom(XmlReader reader)
             return reader.ReadElementAs<Angle>();
         }
 
+        /// <summary>
+        /// Gets the sine of the angle
+        /// </summary>
+        /// <returns>The sine of the angle</returns>
+        public double Sin()
+        {
+            return Math.Sin(this.Radians);
+        }
+
+        /// <summary>
+        /// Gets the cosine of the angle
+        /// </summary>
+        /// <returns>The cosine of the angle</returns>
+        public double Cos()
+        {
+            return Math.Cos(this.Radians);
+        }
+
+        /// <summary>
+        /// Gets the tangent of the angle
+        /// </summary>
+        /// <returns>The tangent of the angle</returns>
+        public double Tan()
+        {
+            return Math.Tan(this.Radians);
+        }
+
+        /// <summary>
+        /// Normalizes an angle to be constrained within start and end.
+        /// </summary>
+        /// <param name="start">The start angle</param>
+        /// <param name="end">The end angle</param>
+        /// <returns>The normalized angle</returns>
+        public Angle Normalize(Angle start, Angle end)
```

Comment:
This is often called clamp

Reply:
shrug I have heard both terms with regards to angles.  If you prefer Clamp I don't mind either way